### ISSUE TITLE
After searching, how to play the searched song?

### ISSUE THREAD

**Issue by kohane27** [aome510/spotify-player#472]
![2024-06-21-07-38-22](https://github.com/aome510/spotify-player/assets/57322459/bbd5f2b7-2eb6-477b-9351-9fe46fbe892e)

Apologies for this dumb question. I've been trying to figure this out for hours.

After searching, how to play the searched song?

I didn't remap any keys and pressing `Tab` or `Shift+Tab` does not do anything. I just can't move the focus from the search input field back to the "previous window". I tried the following keymap:

```toml
[[keymaps]]
command = "FocusPreviousWindow"
key_sequence = "C-h"
```

`C-h` is definitely working because I could trigger the help menu, also it moves the windows in the `LibraryPage`. But after the remap it still does not focus the previous window.

version: spotify_player 0.18.1

Any input is much appreciated. Thank you.

**Comment by kohane27** [comment]
Apologies for the noise. I figured it out.
Turned out it's the bug https://github.com/aome510/spotify-player/issues/436
With only one search result I thought it's not focusing search result UI: it did; it just did not play the currently selected song (i.e., the bug above).
After building from source the bug is gone and hence my problem is gone too. Sorry and thank you!

**Comment by aome510** [comment]
#436 fix should be included in the latest version `v0.18.2`
